patroni/validator.py

```diff
@@ -758,6 +758,25 @@ def assert_(condition: bool, message: str = "Wrong value") -> None:
     assert condition, message
 
 
+class ListValidator(object):
```

Comment:
I would actually call in `EnumValidator`.
But, it will not correctly work for `wal_log_hints` and `hot_standby`.
`'on'`, `'true'`, `'1'`, `1`, `True` -- these are all valid values for them.

Reply:
Yes, something like `boolean_is_true_validator()`. it could be defined right in patroni/postgresql/config.py